test/tests/performance/pbs_jobperf.py

```diff
@@ -95,6 +95,19 @@ def wait_for_job_finish(self, jobiter=True):
         self.server.expect(JOB, {'job_state=F': total_jobs}, extend='x',
                            interval=20, trigger_sched_cycle=False)
 
+    def delete_jobs_per_user(self, users, num_users):
+        """
+        Delete jobs faster by providing more job id's at once
+        """
+        bin_path = str(os.path.join(
+            self.server.pbs_conf['PBS_EXEC'], 'bin'))
+        qdel = str(os.path.join(bin_path, 'qdel'))
+        qdel = qdel + ' -W force'
+        for u in range(0, num_users):
+            cmd = qdel + " `" + str(os.path.join(bin_path, 'qselect'))
```

Comment:
If you choose to use this function move this line out of the loop and it does not change.